.github/actions/build/action.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Build'
+runs:
+  using: "composite"
```

Comment:
Interesting, I've never tried this before, but it makes sense

Reply:
> ```yaml
>  name: 'Build'
>  runs:
>    using: "composite" 
> ```
>
> Interesting, I've never tried this before, but it makes sense

I thought it was the only possible way. Is there some other way it can be done that’s more typical?

Reply:
No, I think in the past I just did more copy/paste. This seems like a better way